src/base-styles/helperClasses/animations/transitions.css

```diff
@@ -0,0 +1,18 @@
+/*---
+title: Transitions
+name: transitions
+section: Animation Helpers
+---
+
+Adds standard transitions
+
+class          | description
+-------------- | ---------------------------------------------------------
+`trsn-default` | transitions all properties to default duration and timing
+*/
+
+.trsn-default {
```

Comment:
for transitions, we may want to use more descriptive names like...

- `.transition--short`
- `.transition--medium--easeOut`
- `.transition--flash`
etc

Reply:
right. Would we want to do that now? I sort of added it this way, as I feel like we don't really have an idea what these different permutations should be.. and what should be short, medium, long, and if those would even be used. So essentially, I'm thinking we could just use `transition--default` and then build upon that if a designer specifically asks for a different timeout. I think maybe putting something about this pattern in some comments might be a nice idea though, when we need to cross that bridge.

Reply:
Would be nice to make the decision later, but changing class names would be a breaking change. Just double checking this to make sure our naming scheme here scales.

Does the concept of a "default" transition make sense across the whole system? That any property being transitioned would fit a default delay, duration, and timing function?

If so, let's keep `transition--default`. If not, let's name it something like `transition--shortEase`.

Reply:
Here's my concern with `default` as a name:

class       | what it looks like it does
----------- | ------------------------------------
`--flashLinear` | super fast linear transition
`--bounceInFast`    | bounces in fast
`--easeOutSlow`    | for slowly fading an element out or something
`--default`    | ???

To be fair, we solved this problem with things like `fontSize` by making `fontSize--default` do the same thing as `fontSize--m`.

We can always map the `--default` variant to a more descriptive name if we need to scale these classes.

Reply:
yeah exactly. I think you have the right thought, but we're not in the right phase to really think about multiple transitions... it's one of those things where we'd add it, but if designers or engineers don't use it, then it's wasted. I'd rather someone ask for a different transition at some point, and then we could even deprecate transition--default in a major FDS release, but I'd rather do that than create a taxonomy that either isn't approved by design, or take the time "to" approve it.